src/rust/lib/graph-editor/src/lib.rs

```diff
@@ -1841,8 +1845,8 @@ fn new_graph_editor(world:&World) -> GraphEditor {
 
     // === Move Edges ===
     detached_edge           <- any(&inputs.some_edge_targets_detached,&inputs.some_edge_sources_detached);
-    cursor_pos_on_detach    <- cursor.frp.position.sample(&detached_edge);
-    edge_refresh_cursor_pos <- any (cursor_pos_on_detach,cursor.frp.position);
+    cursor_pos_on_detach    <- mouse_pos_in_scene.sample(&detached_edge);
+    edge_refresh_cursor_pos <- any (cursor_pos_on_detach,mouse_pos_in_scene);
```

Comment:
Please be very careful about changing the logic! We were using cursor position, now you are using mouse position. So it will not work when snapping ports (cursor moves then, mouse not).